components/authz-service/storage/postgres/migration/legacy/iam.go

```diff
@@ -638,3 +654,12 @@ func convertV1Action(action string, resource string) ([]string, error) {
 		return nil, fmt.Errorf("could not parse V1 action: %s", action)
 	}
 }
+
+func includes(terms []string, termToCheck string) bool {
```

Comment:
Minor potential refactor: I don't see you using this anywhere else, which makes me wonder if we could rename it and push the `singleTermsToMigrate` into the function. Then we could call something like `shouldSkipTermMigration(term)`.

NBD tho since we wanna 🚢